src/api-reference/user-provisioning/spend/v4.spend-user-get-started.md

```diff
@@ -18,6 +18,7 @@ Callers need to acquire the scope listed below in order to access the following
 * [Approver](./v4.spend-user-extension-schema.html#approver-extension-schema)
 * [Delegate](./v4.spend-user-extension-schema.html#delegate-extension-schema)
 * [UserPreference](./v4.spend-user-extension-schema.html#user-preference-extension-schema)
+* [InvoicePreference](v4.spend-user-extension-schema.html#invoice-preference-extension-schema)
```

Comment:
Missing a period in front of v4 line 21? 🙂 (.v4.spend-user-extension-schema.html#invoice-preference-extension-schema)